scene/gui/control.h

```diff
@@ -190,6 +190,13 @@ class Control : public CanvasItem {
 		Vector2 scale = Vector2(1, 1);
 		Vector2 pivot_offset;
 
+		Vector2 render_offset;
+		bool render_offset_relative_to_size = true;
+		Vector2 render_scale = Vector2(1, 1);
+		real_t render_rotation = 0.0;
+		Vector2 render_transform_pivot = Vector2(0, 0);
```

Comment:
```suggestion
		Vector2 render_transform_pivot;
```
Unnecessary default value